client/src/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import icon from "./icon.png";
+
+import "./header.css";
+
+
+
+function Header(){
```

Comment:
<img width="1349" alt="Screen Shot 2021-01-31 at 11 17 03 AM" src="https://user-images.githubusercontent.com/13464239/106375901-3fc4ec80-63b6-11eb-8195-2ab71f39bfb7.png">

As you can see, the items are not vertiically aligned. Also please check the size of the text and logo with the design. you should avoid copying the values for width/height from design (as design is created for 1920 x 1080 resolution as discussed in yesterday's call)